README.md

```diff
@@ -114,13 +114,14 @@ A maintained and comprehensive list of awesome Fuel resources!
 |  Microchain Systems | DEX | Stealth | [microchain](https://microchain-landing.netlify.app/)  |  [@MicrochainSys](https://twitter.com/MicrochainSys) | --  |
 |  Unic | NFT Marketplace |  Stealth | --  | [@0xUNIC](https://twitter.com/0xUNIC)  | --  |
 |  Acumen Finance | Lending | Building |  [acumen.network](https://acumen.network/) |  [@AcumentOfficial](https://twitter.com/acumenofficial) | --  |
-|  Elix Finance | DEX |  Stealth | [elix.finance](https://elixfinance.github.io/interface/)  | --  | --  |
+|  Elix Finance | DEX |  Stealth | [elix.finance](https://elixfinance.github.io/interface/)  | [@ElixFinance](https://twitter.com/elixfinance) | --  |
```

Comment:
The website link is actually broken. Also, I don't think that are "Stealth" anymore.
```suggestion
|  Elix Finance | DEX |  Building | [elix.finance](https://elix.finance/)  | [@ElixFinance](https://twitter.com/elixfinance) | --  |
```